docs/modules/operation/pages/deep-dive/admin/webui/trends.adoc

```diff
@@ -60,95 +61,100 @@ The following sample defines a Trend chart for displaying nodes with ongoing out
 </trend-definition>
 ----
 
-<1> title of the Trend chart, see below for supported variable substitutions
-<2> subtitle of the Trend chart, see below for supported variable substitutions
-<3> defines whether the chart is visible by default
-<4> icon for the chart, see https://getbootstrap.com/docs/4.1/extend/icons/[Icons] for viable options
-<5> options for inline chart, see http://omnipotent.net/jquery.sparkline/#common[jQuery Sparklines] for viable options
-<6> the description link
-<7> the description text, see below for supported variable substitutions
-<8> the SQL statement for querying the chart's values
-
+<1> The title of the Trend chart.
+See below for supported variable substitutions.
+<2> Subtitle of the Trend chart.
+See below for supported variable substitutions.
+<3> The visible flag.
+This defines whether the chart is visible by default.
+<4> The icon for the chart.
+See https://getbootstrap.com/docs/4.1/extend/icons/[Icons] for options.
+<5> Attributes defining the inline chart.
+See http://omnipotent.net/jquery.sparkline/#common[jQuery Sparklines] for options.
+<6> The description link.
+<7> The description text.
+See below for supported variable substitutions.
+<8> The SQL statement that queries the chart's values.
++
 IMPORTANT: Don't forget to limit the SQL query's return values!
```

Comment:
Why? Too many results?

Reply:
I'd assume so, but you're right - clarification would be good here.